sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_LROPolling.md

```diff
@@ -0,0 +1,149 @@
+# Polling the Long Running Operations
+This sample demonstrates how to use polling for the Long Running Operations using the Text Analytics client.
+
+## Creating a `TextAnalyticsClient`
+
+To create a new `TextAnalyticsClient` to run analyze operation for a document, you need a Text Analytics endpoint and credentials.  You can use the [DefaultAzureCredential][DefaultAzureCredential] to try a number of common authentication methods optimized for both running as a service and development.  In the sample below, however, you'll use a Text Analytics API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:TextAnalyticsSample4CreateClient
+var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Polling using `WaitForCompletionAsync()`
+
+To run analyze operation in multiple documents, call `StartAnalyzeOperationBatchAsync` on an `IEnumerable` of strings.  The result is a Long Running operation of type `AnalyzeOperation` which polls for the results from the API.
```

Comment:
I don't think this is needed. Maybe emphasize again that Analyze is used as an example